magento/mod.ts

```diff
@@ -87,6 +86,7 @@ export default function App(props: Props): App<Manifest, State> {
       clientAdmin,
       maxInstallments,
       minInstallmentValue,
+      baseUrl
```

Comment:
se for passar tudo pra state passa so o state, se possível 

return {
    manifest,
    state,
    }